### PR TITLE
fix: cucushift-installer-rehearse-aws-ipi-disconnected-private

### PR DESCRIPTION
**Pull request by tolusha**
Signed-off-by: Anatolii Bazko <abazko@redhat.com>

Fix `workflow-launch cucushift-installer-rehearse-aws-ipi-disconnected-private 4.x` for the clusterbot.
Based on thread https://app.slack.com/client/T027F3GAJ/CNHC2DK2M/thread/CNHC2DK2M-1666597428.268819

### CODE REVIEW

**Comment by tolusha** [comment]
@jhou1 
@liangxia 
Could you have a look pls?

File: ci-operator/step-registry/aws/provision/cco-manual-users/static/aws-provision-cco-manual-users-static-commands.sh [hunk 1]
```diff
@@ -101,7 +101,8 @@ resources_d=`mktemp -d`
 credentials_requests_files=`mktemp`
 echo "extracting CR from image $RELEASE_IMAGE_LATEST"
 oc version --client
-cmd="oc adm release extract ${RELEASE_IMAGE_LATEST} --credentials-requests --cloud=aws --to '$cr_yaml_d'"
+REPO=$(oc -n ${NAMESPACE} get is release -o json | jq -r '.status.publicDockerImageRepository')
+cmd="oc adm release extract ${REPO}:latest --credentials-requests --cloud=aws --to '$cr_yaml_d'"
```
Comment (jianlinliu) [inline ci-operator/step-registry/aws/provision/cco-manual-users/static/aws-provision-cco-manual-users-static-commands.sh:105]: The step works well in our daily jobs. E.g: https://gcsweb-qe-private-deck-ci.apps.ci.l2s4.p1.openshiftapps.com/gcs/qe-private-deck/logs/periodic-ci-openshift-openshift-tests-private-release-4.12-amd64-stable-aws-ipi-disconnected-private-sno-p2-f7/1596920313080188928/artifacts/aws-ipi-disconnected-private-sno-p2-f7/aws-provision-cco-manual-users-static/build-log.txt
```
extracting CR from image registry.build03.ci.openshift.org/ci-op-bl82zwgb/release@sha256:ac7dc25ce902c845826b7db255341a25c2f8030a1567d8bd54ea83505974c024
Client Version: v4.2.0-alpha.0-1642-g854f807
Kustomize Version: v4.5.7
Running Command: oc adm release extract registry.build03.ci.openshift.org/ci-op-bl82zwgb/release@sha256:ac7dc25ce902c845826b7db255341a25c2f8030a1567d8bd54ea83505974c024 --credentials-requests --cloud=aws --to '/tmp/tmp.qtKY4Yq6tT'
Warning: the default reading order of registry auth file will be changed from "${HOME}/.docker/config.json" to podman registry config locations in the future version of oc. "${HOME}/.docker/config.json" is deprecated, but can still be used for storing credentials as a fallback. See https://github.com/containers/image/blob/main/docs/containers-auth.json.5.md for the order of podman registry config locations.
Extracted release payload created at 2022-11-21T12:23:02Z
```
The looks good to me. The only concern is ensure `${REPO}:latest` is pointed to the same image as `${RELEASE_IMAGE_LATEST}` like what we did before.

To confirm that, can you help append two more debugging lines here:
```
oc image info ${RELEASE_IMAGE_LATEST}  || true
oc image info ${REPO}:latest || true
```

Comment (tolusha) [reply ci-operator/step-registry/aws/provision/cco-manual-users/static/aws-provision-cco-manual-users-static-commands.sh:105]: Thank you for checking.
Added debug commands.

**Pull request merged**